core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1749,6 +1772,7 @@ object SnappySession extends Logging {
   private[sql] val ExecutionKey = "EXECUTION"
 
   lazy val isEnterpriseEdition: Boolean = {
+    GemFireCacheImpl.setGFXDSystem(true)
```

Comment:
Same question as above. Why is this early init required. It happens in store boot. Or is it for SmartConnectors?

Reply:
Local mode unit tests is where I saw the problem. Didn't exactly note down the test/scenario.